README.md

```diff
@@ -4,76 +4,123 @@
 
-Open or edit a manifest file *(pom.xml/package.json)* to trigger inline `Component Analysis`. This feature scans your application and provides inline feedback on any dependencies identified with security vulnerabilities.
+If you need a new Snyk token, you can generate a new token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&utm_source=code_ready&code_ready=FF1B53D9-57BE-4613-96D7-1D06066C38C9).
 
-Generate a `Red Hat Dependency Analytics Report` for your application by performing any of the following actions:
-- Right click on a manifest file in the *Vscode File explorer* or *Vscode File editor* and choose the `Red Hat Dependency Analytics Report` option.
-- On an open manifest file, click on the *pie icon* ![icon](icon/report-icon.png) located at the upper right corner in the tab container.
-- On an open manifest file, hoover over a dependency marked by the inline `Component Analysis`, click on *Quick Fix* and choose *Detailed Vulnerability Report* option`.
-- Click on `Open detailed vulnerability report` button inside the vulnerability alert popup message.
-- Click on `Found <NUMBER> vulnerabilities` in the *status bar* bellow.
-- Click on `Dependency analysis in progress` in the *status bar* bellow (only visible when Component Analysis is in progress and not yet complete).
-<br > Check out the [features](#features) section for more details.
+**Red Hat Dependency Analytics Report File Path** :
 
-> **NOTE** `Red Hat Dependency Analytics` is an online service hosted and maintained by Red Hat. This open source software will access only your manifests file(s) to analyse the application dependencies within before providing you with a detailed report.
+Specify the local path to create the Dependency Analytics report file.
+The default path is `/tmp/redhatDependencyAnalyticsReport.html`.
 
 ## Features
 
-1. **Component Analysis**
-	<br >Upon opening or editing a manifest file *(pom.xml/package.json)*, an automated scan will be triggered on your application. This process provides immediate, inline feedback regarding any dependencies that have been detected to have security vulnerabilities. Such dependencies will be appropriately flagged in red and present a short summary when hoovered over. Summary contains the full package name and version, amount of known security vulnerabilities and highest severity status of said vulnerabilities.
-	<br >**Note** a *target* folder will be created in the workspace, used to process pom.xml files. Please add *target* to *.gitignore*.
+- **Component analysis**
+	<br >Upon opening a manifest file, such as a `pom.xml` or `package.json` file, a scan starts the analysis process.
+	The scan provides immediate inline feedback on detected security vulnerabilities for your application's dependencies.
+	Such dependencies are appropriately underlined in red, and hoovering over it gives you a short summary of the security concern.
```

Comment:
same here, _hovering over_?

Reply:
Good catch!  I'll get it fixed.

Reply:
I verified that both misspellings have been fixed.  Thanks for taking a look and providing your feedback!